klippy/extras/output_pin.py

```diff
@@ -45,12 +45,12 @@ def get_status(self, eventtime):
     def cmd_SET_PIN(self, gcmd):
         value = gcmd.get_float('VALUE', minval=0., maxval=self.scale)
         value /= self.scale
-        if value == self.last_value:
-            return
```

Comment:
I'm not sure we want to eliminate this check, as we wouldn't want to send redundant schedule_soft_pwm commands to the micro-controller if the user does not actually request a change.

Reply:
I added it back while also checking for changes of `cycle_time`.